content/en/docs/tasks/administration/client-status-change/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Disable or Enable a client
+weight: 2
+---
+Existing clients can be disabled or enabled by admin user from dashboard.
+
+## Disabling a client using the dashboard
+
+Log into the service using admin credential and click on the _Clients_ link on the left
+navigation bar:
+
+![dashboard](../images/client-status-change-1.png)
+
+From the _Clients_ link, select _Any client_ you want to disable, for example _Test Client_:
+
+![client list](../images/client-status-change-2.png)
+
+To disable the client click on _Disable Client_ button on the bottom of the page:
+
+![disable client button](../images/client-status-change-3.png)
+
+To confirm your choice click on _Disable Client_ button on the modal window:
+
+![disable client modal](../images/client-status-change-4.png)
+
+On success you will get a confirmation message and the _Disable Client_ button will change to _Restore Client_:
+
+![disable client confirmation](../images/client-status-change-5.png)
+
+Using _Restore Client_ button client's status can be change back to _Active_.
```

Comment:
Fix typos: **client status** can be **changed**

Reply:
Thanks. Fixed.